tools/topology/topology2/cavs/cavs-nocodec-mclk-192-bclk-12288-rate-96k-tdm-32-slot-4.conf

```diff
@@ -0,0 +1,374 @@
+<include/common/vendor-token.conf>
+<include/common/tokens.conf>
+<include/common/manifest.conf>
+<include/common/pcm.conf>
+<include/common/pcm_caps.conf>
+<include/common/fe_dai.conf>
+<include/common/route.conf>
+<include/components/host.conf>
+<include/components/dai.conf>
+<include/components/pipeline.conf>
+<include/components/copier.conf>
+<include/dais/ssp.conf>
+<include/dais/dmic.conf>
+<include/dais/pdm_config.conf>
+<include/common/data.conf>
+<include/dais/hw_config.conf>
+<include/pipelines/cavs/passthrough-playback.conf>
+<include/pipelines/cavs/passthrough-capture.conf>
+
+#
+# List of all DAIs
+#
+#SSP Index: 0, Direction: duplex
+Object.Dai {
+	SSP."0" {
+		id 		0
+		direction	"duplex"
+		name		NoCodec-0
+		default_hw_conf_id	0
+		sample_bits		16
+		quirks			"lbm_mode"
+
+		Object.Base.hw_config."SSP0" {
+			id	0
+			mclk_freq	19200000
+			bclk_freq	12288000
```

Comment:
right there this is asking for trouble. Why on earth would you select different clock sources for bclk and mclk?

Reply:
@lgirdwood I think this is diable without any changes in the compiler. There are hooks available already to have a set of defines which can be platform specific and we can add constraints that restrict the settings to the available values per platform. I can guide @udaymb to add this and use it

Reply:
@plbossart - There is ask for the support for multiple mclk and bclk, hence created this PR

@ranj063 - Thanks for the inputs, based on the new nocodec.conf, removed multiple files and made the relevant changes in CMakeLists and nocodec file

Reply:
Please check the new changes and share your inputs